advertise/src/main/java/net/androidweekly/advertise/AdvertiseFragment.kt

```diff
@@ -0,0 +1,50 @@
+package net.androidweekly.advertise
+
+
+import android.os.Bundle
+import android.view.View
+import android.webkit.WebView
+import android.webkit.WebViewClient
+import android.widget.ProgressBar
+import net.androidweekly.core.fragments.BaseFragment
+import net.androidweekly.core.utils.android.bindView
+
+/**
+ * Project: Android Weekly
+ * Created: November 25, 2019
+ *
+ * @author Khaled Mohammad
+ */
+class AdvertiseFragment : BaseFragment() {
+
+    private val advertiseWebView: WebView? by bindView(R.id.web_view_fragment_advertise)
+    private val progressBar: ProgressBar? by bindView(R.id.progress_bar_fragment_advertise)
```

Comment:
Please use `findViewById` here and make sure if you put the fragment in the back stack no leaks will happen